src/functionModules/components/FileListerModule.ts

```diff
@@ -11,37 +13,36 @@ import { FlashType } from '../../models/Flash'
 
 export default class FileListerModule {
 
-  public static async list(web3Module: Web3Module,
-                           flashesModule: FlashesModule,
-                           newListingModule: NewListingModule,
-                           uploadModule: UploadModule) {
+  public static async list(appStore: Store<any>) {
 
+    const newListingModule = getModule(NewListingModule, appStore)
     try {
       newListingModule.setPercentComplete(50)
 
       // TODO: validate the listing?
       await ListingModule.postListing(
```

Comment:
Nit: `ListingModule` -> `ListingContractModule` I think may be more semantic (not to be confused with vuexModules)

Reply:
not a nit. needs to be replaced.  keep catching them all like pokemon.